server/src/test/scala/org/http4s/netty/server/ServerTest.scala

```diff
@@ -142,6 +142,22 @@ abstract class ServerTest extends IOSuite {
         .map(_.merge)
     }
   }
+
+//  test("H2 Prior Knowledge is supported") {
```

Comment:
Need a better way to test this. This was here so I could test manually with curl which supports prior knowledge via the flag --http2-prior-knowledge.

Reply:
it appears that jetty-client can support http/2 over plaintext, so that might be a way forward.
we have a client here: https://github.com/http4s/http4s-jetty